ext/com_dotnet/com_com.c

```diff
@@ -251,15 +251,15 @@ PHP_METHOD(com, __construct)
 			BSTR name;
 
 			if (SUCCEEDED(ITypeLib_GetDocumentation(TL, -1, &name, NULL, NULL, NULL))) {
-				typelib_name = php_com_olestring_to_string(name, &typelib_name_len, obj->code_page);
+				zend_string *typelib_str = php_com_olestring_to_string(name, obj->code_page);
 
-				if (NULL != php_com_cache_typelib(TL, typelib_name, typelib_name_len)) {
+				if (NULL != php_com_cache_typelib(TL, ZSTR_VAL(typelib_str), ZSTR_LEN(typelib_str))) {
 					php_com_import_typelib(TL, mode, obj->code_page);
 
 					/* add a reference for the hash */
 					ITypeLib_AddRef(TL);
 				}
-				efree(typelib_name);
+				zend_string_release_ex(typelib_str, /* persistant */ false);
```

Comment:
persistent

Reply:
Indeed >_> Just been writing French basically in php-src.